app/controllers/users/devise_authy_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Users::DeviseAuthyController < Devise::DeviseAuthyController
+  layout "application"
+
+  protected
+  
+  def after_authy_enabled_path_for(resource)
+  end
+
+  def after_authy_verified_path_for(resource)
+    if resource.sign_in_count > 1
+      root_path
+    else
+      onboarding_success_path
+    end
```

Comment:
Minor style thing:
```suggestion
    return root_path if resource.sign_in_count > 1

    onboarding_success_path
```
or
```suggestion
    resource.sign_in_count > 1 ? root_path : onboarding_success_path
```